dev-resources/private/test/src/cljc/foo/bar/core.cljc

```diff
@@ -0,0 +1,16 @@
+(ns foo.bar.core
+  (#?(:clj  :require
+      :cljs :require-macros)
+    [foo.bar.macros]))
```

Comment:
Is the same as [here](http://blog.fikesfarm.com/posts/2015-06-19-portable-macro-musing.html)